### PR TITLE
cleanout unused variables

### PR DESCRIPTION
**Pull request by pindge**
## Scope
- delete `not in use` variables
- split variables by projects
- move automated reporting variable into its own folder
- create `hhrs` dag folder and move all its dags into that folder for organisation

### CODE REVIEW

File: dags/infra/projects/automated_reporting.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# variables
+"""
+from airflow.models import Variable
+
+# automated-reporting
+COP_API_REP_CREDS = "copernicus_api_password"  # not in use
+
+AWS_STATS_SECRET = Variable.get(
+    "aws_stats_secret", default_var="aws-stats"
+)  # not used anywhere
```
Comment (pindge) [inline dags/infra/projects/automated_reporting.py:12]: @tmcadam, @ramagopr all the variables in this file are not used anywhere, are you happy for me to delete all of them? I have moved the ones in use to `automated reporting` folder

File: dags/infra/projects/hnrs.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+)
+
+SECRET_HNRS_DC_WRITER_NAME = Variable.get(
+    "db_hnrs_dc_writer_secret", default_var="hnrs-dc-writer"
+)
+
+WATERBODIES_DEV_USER_SECRET = Variable.get(
+    "waterbodies_dev_user_secret", default_var="waterbodies-dev-user-creds"
+)
+ELVIS_DEV_USER_SECRET = Variable.get(
```
Comment (pindge) [inline dags/infra/projects/hnrs.py:18]: @supermarkion  this variables isnt used anywhere, do you still need it?

Comment (supermarkion) [reply dags/infra/projects/hnrs.py:18]: Hi, @pindge, thanks for your message. Yes, we still need it: https://github.com/GeoscienceAustralia/dea-airflow/blob/develop/dags/k8s_waterbodies_conflux.py#L60. In short, this AWS Cred has permission to dump files to this folder (also only this folder): http://dea-public-data-dev.s3-website-ap-southeast-2.amazonaws.com/?prefix=projects/WaterBodies/.

Comment (supermarkion) [reply dags/infra/projects/hnrs.py:18]: The `ELVIS_DEV_USER_SECRET` can be removed. We already switch to another AWS role (https://github.com/GeoscienceAustralia/dea-airflow/blob/develop/dags/k8s_index_elvis_lidar_dem_test_loop.py#L132) in its index pipeline.

File: dags/infra/projects/automated_reporting.py (hunk 1 above)
Comment (ramagopr) [reply dags/infra/projects/automated_reporting.py:12]: looks good

**Review by ramagopr — APPROVED** [review]

**Pull request merged**